src/datachain/data_storage/id_generator.py

```diff
@@ -33,6 +33,15 @@ def init(self) -> None:
     def cleanup_for_tests(self):
         """Cleanup for tests."""
 
+    def close(self) -> None:
+        """Closes any active database connections."""
+
+    def close_on_exit(self) -> None:
+        """Closes any active database or HTTP connections, called on exit or for test
```

Comment:
[C] could mention session here as it appears that is where close_on_exit is currently called. I.e. - on `Session` exit

Reply:
same for the other `close_on_exit` methods

Reply:
Yep, changed the comment to reference Session exit.